libs/sandboxes-service-api-client/src/ApiClientConfiguration.php

```diff
@@ -14,18 +13,20 @@ class ApiClientConfiguration
 {
     private const DEFAULT_BACKOFF_RETRIES = 10;
 
-    private const DEFAULT_USER_AGENT = 'Keboola Sandboxes Service API PHP Client';
-
     /**
      * @param int<0, max> $backoffMaxTries
      */
     public function __construct(
-        public readonly string $userAgent = self::DEFAULT_USER_AGENT,
+        public readonly string $baseUrl,
+        public readonly string $storageToken,
+        public readonly string $userAgent,
         public readonly int $backoffMaxTries = self::DEFAULT_BACKOFF_RETRIES,
-        public readonly null|StorageTokenAuthenticator $authenticator = null,
         public readonly null|Closure $requestHandler = null,
         public readonly LoggerInterface $logger = new NullLogger(),
     ) {
+        Assert::minLength($this->baseUrl, 1);
+        Assert::stringNotEmpty($this->storageToken);
```

Comment:
proc jedno je `minLength` a druhy `stringNotEmpty`? Nemelo by to byt stejny?

Reply:
Puvodne tam bylo min length a ja tam pridal to notEmpty, upravim vsechno na notEmpty a podle toho i hlasky v testech